### PR TITLE
Changes related to the rebranding

### PR DESCRIPTION
**Pull request by brunoocasali**
These changes were already merged, but incorrectly so this PR fixes that!

Related to #225

### CODE REVIEW

File: src/Meilisearch/Meilisearch.csproj [hunk 1]
```diff
@@ -5,9 +5,9 @@
         <OutputType>Library</OutputType>
         <PackageId>MeiliSearch</PackageId>
         <Version>0.9.0</Version>
-        <Description>.NET wrapper for meilisearch, an open-source search engine</Description>
+        <Description>.NET wrapper for Meilisearch, an open-source search engine</Description>
         <RepositoryUrl>https://github.com/meilisearch/meilisearch-dotnet</RepositoryUrl>
-        <PackageTags>meilisearch;dotnet;sdk;search-engine;search;instant-search</PackageTags>
+        <PackageTags>Meilisearch;dotnet;sdk;search-engine;search;instant-search</PackageTags>
```
Comment (alallema) [inline src/Meilisearch/Meilisearch.csproj]: It's ok to modify this one ?

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -107,7 +107,7 @@ getting_started_add_documents_md: |-
   using System.Threading.Tasks;
   using System.Collections.Generic;
 
-  namespace meilisearch_demo
+  namespace Meilisearch_demo
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:110]:
```suggestion
  namespace meilisearch_demo
```

File: src/Meilisearch/Meilisearch.csproj (hunk 1 above)
Comment (curquiza) [reply src/Meilisearch/Meilisearch.csproj]: Ok to keep! :)

**Review by bidoubiwa — DISMISSED** [review]
LGTM !

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -22,5 +22,5 @@ If applicable, add screenshots or logs to help explain your problem.
 
 **Environment (please complete the following information):**
 - OS: [e.g. Debian GNU/Linux]
-- meilisearch version: [e.g. v.0.20.0]
-- meilisearch-dotnet version: [e.g v0.6.0]
+- Meilisearch version: [e.g. v.0.20.0]
+- Meilisearch-dotnet version: [e.g v0.6.0]
```
Comment (bidoubiwa) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- meilisearch-dotnet version: [e.g v0.6.0]
```

Should we keep the package name lowercase?

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (brunoocasali) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, definitely! ;)

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**